scipy/_lib/_testutils.py

```diff
@@ -76,7 +76,9 @@ def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
         pytest_args = ['--showlocals', '--tb=short']
 
         if doctests:
-            raise ValueError("Doctests not supported")
+            # raise ValueError("Doctests not supported")
+            # XXX: maybe do not ignore `scipy._lib._disjoint_set.DisjointSet` etc
+            pytest_args += ["--doctest-modules", "--ignore=scipy/interpolate/_interpnd_info.py", "--ignore=scipy/_lib/"]
```

Comment:
Looks like we're fine to restrict the `_lib` ignores to just the git submodules.

```suggestion
            pytest_args += [
                "--doctest-modules",
                "--ignore=scipy/interpolate/_interpnd_info.py",
                "--ignore=scipy/_lib/array_api_compat",
                "--ignore=scipy/_lib/highs",
                "--ignore=scipy/_lib/unuran",
            ]
```

Reply:
Great! Okay, will make this change.